plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -347,18 +346,20 @@ func (l *windowsEventLog) getRecords(handles []EvtHandle) (records []*windowsEve
 			//old SSM agent Windows format
 			var recordMessage eventMessage
 			if err = xml.Unmarshal(descriptionBytes, &recordMessage); err != nil {
-				return nil, err
+				log.Printf("I! [windows_event_log] UTF16ToUTF8Bytes() err %v", err)
+				continue
 			}
 
 			newRecord.System.Description = recordMessage.Message
 		default:
-			return nil, fmt.Errorf("format %s is not recognized", l.renderFormat)
+			log.Printf("I! [windows_event_log] l.renderFormat is not recognized, %s", l.renderFormat)
```

Comment:
This file is part of the `package wineventlog`.
It makes sense to me to change all the logging messages to have `[wineventlog]` instead of `[windows_event_log]`.
I didn't want to clutter this PR anymore though.

Reply:
Frankly I'm fine with including it in this change. That or a followup PR that does that change. I agree that the qualifier of the log message should match the package

Reply:
Fixed.